src/store/__tests__/reducer.test.ts

```diff
@@ -4,8 +4,7 @@ import { Reducer } from '@reduxjs/toolkit'
 describe('reducer', () => {
   it('should define app reducers', () => {
     const reducer = createReducer() as Reducer<any, any>
-    const state = { a: 1 }
```

Comment:
Ahhh `Unexpected key "a" found in previous state received` from https://github.com/oasisprotocol/oasis-wallet-web/pull/800